code/modules/assembly/signaler.dm

```diff
@@ -29,6 +29,18 @@ GLOBAL_LIST_EMPTY(remote_signalers)
 	. = ..()
 	. += "The power light is [receiving ? "on" : "off"]"
 
+/obj/item/assembly/signaler/AltClick(mob/user)
+	activate()
```

Comment:
Can you add a to_chat message here, so the user knows they purposely activated the signal?

Reply:
There are beeps on the chat once the signal is sent, do you think a to_chat should be added to the Activate() proc?

Reply:
Yeah, because its not clear that the beeps were them if they somehow accidently alt-clicked it.

Reply:
Added with the next commit, think this is a good place for it?

Reply:
I think its better alone in the altclick proc, since its much harder to accidently send it with the UI open

Reply:
I would have to add the same cooldown that the signal activation has only for this to_chat here, should i go for it?